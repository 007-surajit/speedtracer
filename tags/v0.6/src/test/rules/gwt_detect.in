hintlet_gwt_detect.js

# Host page
{"type":23,"time":1.0,"data":{"resourceId":"0-13","mimeType":"text/html","headers":{"Date":"Tue, 24 Nov 2009 15:43:52 GMT","Content-Length":"393","Last-Modified":"Tue, 10 Nov 2009 16:29:03 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"5d5347-189-47806cee391c0\"","Content-Type":"text/html","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp.html"},"sequence":6}

# GWT selection script
# HINT
{"type":23,"time":2.0,"data":{"resourceId":"0-14","mimeType":"application/x-javascript","headers":{"Date":"Tue, 24 Nov 2009 15:43:53 GMT","Content-Length":"6347","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"f892ed-18cb-47806cf115880\"","Content-Type":"application/x-javascript","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/SpeedTracerDemoApp.nocache.js"},"sequence":14}

# ???
{"type":23,"time":3.0,"data":{"resourceId":"0-15","mimeType":"text/html","headers":{},"responseCode":0,"isCached":false,"url":"about:blank"},"sequence":19}

# Stylesheet
{"type":23,"time":4.0,"data":{"resourceId":"0-16","mimeType":"text/css","headers":{"Date":"Tue, 24 Nov 2009 15:30:36 GMT","Content-Length":"25214","Last-Modified":"Tue, 10 Nov 2009 16:29:07 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"aaad5f-627e-47806cf209ac0\"","Content-Type":"text/css","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/gwt/standard/standard.css"},"sequence":27}

# GWT Initial download (strongname file) 
# 
{"type":23,"time":5.0,"data":{"resourceId":"0-17","mimeType":"text/html","headers":{"Date":"Tue, 24 Nov 2009 15:30:36 GMT","Content-Length":"134273","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"1b5892b-20c81-47806cf115880\"","Content-Type":"text/html","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/DA85A01950553A52FB8150E588B71041.cache.html"},"sequence":31}

#image
{"type":23,"time":6.0,"data":{"resourceId":"0-18","mimeType":"image/png","headers":{"Date":"Tue, 24 Nov 2009 15:43:53 GMT","Content-Length":"13577","Last-Modified":"Tue, 10 Nov 2009 16:29:07 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"662ca9-3509-47806cf209ac0\"","Content-Type":"image/png","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/timely_tracey.png"},"sequence":47}

#image
{"type":23,"time":7.0,"data":{"resourceId":"0-19","mimeType":"image/png","headers":{"Date":"Tue, 24 Nov 2009 15:43:54 GMT","Content-Length":"13652","Last-Modified":"Tue, 10 Nov 2009 16:29:07 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"662ca8-3554-47806cf209ac0\"","Content-Type":"image/png","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/pony.png"},"sequence":53}

# new selection script
{"type":23,"time":9.0,"data":{"resourceId":"0-14","mimeType":"application/x-javascript","headers":{"Date":"Tue, 24 Nov 2009 15:43:53 GMT","Content-Length":"6347","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"f892ed-18cb-47806cf115880\"","Content-Type":"application/x-javascript","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/SpeedTracerDemoApp2.nocache.js"},"sequence":14}

# new strongname 
# No hint, because there has been no page transition.
{"type":23,"time":10.0,"data":{"resourceId":"0-17","mimeType":"text/html","headers":{"Date":"Tue, 24 Nov 2009 15:30:36 GMT","Content-Length":"134273","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"1b5892b-20c81-47806cf115880\"","Content-Type":"text/html","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/DA85A01950553A50008150E588B71041.cache.html"},"sequence":31}

# page transition
{"type":16,"time":11.0,"data":{"url":"http://www.example.com"}}

# new selection script
# HINT
{"type":23,"time":12.0,"data":{"resourceId":"0-14","mimeType":"application/x-javascript","headers":{"Date":"Tue, 24 Nov 2009 15:43:53 GMT","Content-Length":"6347","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"f892ed-18cb-47806cf115880\"","Content-Type":"application/x-javascript","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/SpeedTracerDemoApp3.nocache.js"},"sequence":14}

# new strongname 
# 
{"type":23,"time":13.0,"data":{"resourceId":"0-17","mimeType":"text/html","headers":{"Date":"Tue, 24 Nov 2009 15:30:36 GMT","Content-Length":"134273","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"1b5892b-20c81-47806cf115880\"","Content-Type":"text/html","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/DA85A01950553A50008150E588B71041.cache.html"},"sequence":31}

# page transition
{"type":16,"time":14.0,"data":{"url":"http://www.example.com"}}

# new selection script
# No hint, caching is turned OFF on this file
{"type":23,"time":15.0,"data":{"resourceId":"0-14","mimeType":"application/x-javascript","headers":{"Cache-Control": "no-cache","Date":"Tue, 24 Nov 2009 15:43:53 GMT","Content-Length":"6347","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"f892ed-18cb-47806cf115880\"","Content-Type":"application/x-javascript","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/SpeedTracerDemoApp3.nocache.js"},"sequence":14}

# new strongname 
# 
{"type":23,"time":16.0,"data":{"resourceId":"0-17","mimeType":"text/html","headers":{"Date":"Tue, 24 Nov 2009 15:30:36 GMT","Content-Length":"134273","Last-Modified":"Tue, 10 Nov 2009 16:29:06 GMT","Server":"Apache/2.0.55 (Ubuntu)","ETag":"\"1b5892b-20c81-47806cf115880\"","Content-Type":"text/html","Accept-Ranges":"bytes"},"responseCode":200,"isCached":false,"url":"http://www.example.com/timely/SpeedTracerDemoApp/DA85A01950553A50008150E588B71041.cache.html"},"sequence":31}
