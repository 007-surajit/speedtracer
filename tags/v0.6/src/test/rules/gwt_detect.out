{"hintletRule":"GWT Application Detection","timestamp":2,"description":"GWT selection script '.nocache.js' file should be set as non-cacheable","refRecord":2,"severity":1}
{"hintletRule":"GWT Application Detection","timestamp":12,"description":"GWT selection script '.nocache.js' file should be set as non-cacheable","refRecord":11,"severity":1}
